fulp_modules/features/antagonists/infiltrators/infil_items.dm

```diff
@@ -238,9 +238,88 @@
     return TRUE
 
 /mob/living/silicon/robot/emag_act(mob/user, obj/item/card/emag/emag_card)
-    . = ..()
-    if(istype(emag_card, /obj/item/card/emag/silicon_hack))
-        var/obj/item/card/emag/silicon_hack/hack_card = emag_card
-        hack_card.use_charge(user)
-        playsound(src, 'sound/machines/beep.ogg', 50, FALSE)
-        return
+	. = ..()
+	if(!istype(emag_card, /obj/item/card/emag/silicon_hack))
+		return
+	if(!opened)
+		return
+	var/obj/item/card/emag/silicon_hack/hack_card = emag_card
+	hack_card.use_charge(user)
+	playsound(src, 'sound/machines/beep.ogg', 50, FALSE)
+
+/obj/item/missile_disk
+	name = "Missile Disk"
+	desc = "Used to store the coordinates of the station."
+	icon = 'fulp_modules/features/antagonists/infiltrators/icons/infils.dmi'
+	icon_state = "missiledisk"
+	w_class = WEIGHT_CLASS_TINY
+	item_flags = NOBLUDGEON
+	///Does the disk have the station coordinates?
+	var/stored = FALSE
+
+/obj/item/missile_disk/afterattack(atom/movable/victim, mob/living/carbon/human/user, proximity)
+	. = ..()
+	if(!proximity)
+		return
+	if(!istype(victim, /obj/machinery/computer/communications))
+		return
+
+	to_chat(user, span_warning("Downloading station coordinates..."))
+	if(!do_after(user, 8 SECONDS))
+		return
+	playsound(src, 'sound/machines/beep.ogg', 50, FALSE)
+	to_chat(user, span_warning("Station coordinates successfully downloaded!"))
+	stored = TRUE
+
+/obj/item/missilephone
+	name = "Large Handphone"
+	desc = "Hello? Is this the police?"
```

Comment:
lowercase + dunno if the desc fits too well

Reply:
KUNGGGGGGGGGGGGGG FURY REFERENCE

Reply:
> lowercase + dunno if the desc fits too well

it do